clang/test/SemaSYCL/array-kernel-param-neg.cpp

```diff
@@ -4,6 +4,13 @@
 // an array of non-trivially copyable structs as SYCL kernel parameter or
 // a non-constant size array.
 
+// FIXME: the reason for the missing-expected-error comments is because
```

Comment:
The 'lit' test likely won't be able to catch this, even after the driver works, since the host-compile doesn't run with a -cc1 invocation.

Reply:
I was thinking the same, but I figured someone would want to see where the old diagnostics were being emitted when hooking the integration footer up to the host compile. Do you think there's a better way for me to do that? (I could just replace the old tests with the new tests and whoever does that hookup work can use version control to see the old diagnostic locations, I suppose.)

Reply:
Maybe a 'note' instead of a FIXME?